sample-parent/sample-connector/src/main/resources/icons/README.md

```diff
@@ -0,0 +1,10 @@
+Downloaded from
+https://icons8.com/icon/set/plants/material
+Or created with 
+https://uxwing.com/svg-icon-editor/
```

Comment:
Are those ressources OSS? Need to indicate license...

Reply:
corrected